homeassistant/components/elmax/common.py

```diff
@@ -108,27 +110,18 @@ async def _async_update_data(self):
                 return None
 
         except ElmaxBadPinError as err:
-            _LOGGER.error("Control panel pin was refused")
-            raise ConfigEntryAuthFailed from err
+            raise ConfigEntryAuthFailed("Control panel pin was refused") from err
         except ElmaxBadLoginError as err:
-            _LOGGER.error("Refused username/password")
-            raise ConfigEntryAuthFailed from err
+            raise ConfigEntryAuthFailed("Refused username/password") from err
         except ElmaxApiError as err:
-            raise HomeAssistantError(
-                f"Error communicating with ELMAX API: {err}"
-            ) from err
-        except ElmaxNetworkError as err:
```

Comment:
Is `ElmaxNetworkError` not a valid exception to catch anymore?

Reply:
Yes. It is a network error from which we cannot recover. Therefore the broader handler offered by the framework is ok for me.

Reply:
If it's a known exception we want to catch it explicitly. Only unexpected exceptions should be handled by the broad except handler.

Reply:
Done.